src/components/cqlLibraryList/CqlLibraryList.test.tsx

```diff
@@ -547,13 +622,31 @@ describe("CqlLibrary List component", () => {
       `view/edit-cqlLibrary-button-${cqlLibrary[0].id}`
     );
     userEvent.click(viewEditButton);
+
+    await waitFor(() => {
+      expect(
+        screen.getByTestId(`create-new-version-${cqlLibrary[0].id}-button`)
+      ).toBeInTheDocument();
+    });
     const versionButton = screen.getByTestId(
       `create-new-version-${cqlLibrary[0].id}-button`
     );
     userEvent.click(versionButton);
-    userEvent.click(screen.getByLabelText("Major"));
+
+    await waitFor(() => {
+      expect(screen.getByLabelText("Major")).toBeInTheDocument();
+    });
+    const majorButton = screen.getByLabelText("Major");
+    userEvent.click(majorButton);
+
+    await waitFor(() => {
+      expect(
+        screen.getByTestId("create-version-continue-button")
+      ).toBeInTheDocument();
+    });
+    const continueButton = screen.getByTestId("create-version-continue-button");
     await waitFor(() => {
-      userEvent.click(screen.getByTestId("create-version-continue-button"));
+      userEvent.click(continueButton);
```

Comment:
Might be better to perform the click outside of the 'waitFor' to prevent re-clicks should the following getBy not find anything.

Reply:
updated